src/interfaces/logs.ts

```diff
@@ -36,6 +38,7 @@ export interface LogObject {
   responseTime: number
   alert: Alerts
   notification: Alerts
+  method: Method | undefined
```

Comment:
What is the case that can make `method` is undefined?

Reply:
Let's say that user defined the config using minimal configuration:

```
{
   "probes": [
     {
       "requests": [
         {
           "url": "https://github.com"
         }
       ]
     }
   ]
 }
```

They did not define the request method. So that's why when I didn't put `undefined` in the interface, this happens:

![image](https://user-images.githubusercontent.com/16670475/127275634-cf17c481-9a98-4cfa-ac72-a45f87c9744a.png)